.github/workflows/build-and-test.yaml

```diff
@@ -31,25 +31,40 @@ jobs:
           else
             exit 1
           fi
+
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v2
+
+      - name: Restore dependencies
+        run: |          
+          retries=2
```

Comment:
use the same settings as https://github.com/finbourne/configuration-sdk-extensions-csharp/blob/master/.github/workflows/build-and-test.yaml#L42 - sleep for 1 min and try for 15min

Reply:
I was looking at the original PR and never actually looked at the most recent version of the file...
Will go through an update them all